src/app/routes/podcast/getInitialData/index.js

```diff
@@ -0,0 +1,110 @@
+import path from 'ramda/src/path';
+import pathOr from 'ramda/src/pathOr';
+import fetchPageData from '../../utils/fetchPageData';
+import overrideRendererOnTest from '../../utils/overrideRendererOnTest';
+import getPlaceholderImageUrlUtil from '../../utils/getPlaceholderImageUrl';
+import pathWithLogging, {
+  LOG_LEVELS,
+} from '../../../lib/utilities/logging/pathWithLogging';
+import { RADIO_MISSING_FIELD } from '#lib/logger.const';
+import getEpisodeAvailability, {
+  getUrl,
+} from '#lib/utilities/episodeAvailability';
+import getErrorStatusCode from '../../utils/fetchPageData/utils/getErrorStatusCode';
+import withRadioSchedule from '#app/routes/utils/withRadioSchedule';
+import getRadioService from '../../utils/getRadioService';
+import processRecentEpisodes from '../../utils/processRecentEpisodes';
+
+const DEFAULT_TOGGLE_VALUE = { enabled: false, value: 8 };
+
+const getRadioScheduleData = path(['radioScheduleData']);
+const getScheduleToggle = path(['onDemandRadioSchedule', 'enabled']);
+const getRecentEpisodesToggle = pathOr(DEFAULT_TOGGLE_VALUE, [
+  'recentPodcastEpisodes',
```

Comment:
Changed the toggle initially - can revert as there is a different ticket for this work